### PR TITLE
Add template endpoints

### PR DESCRIPTION
**Pull request by devjavi**
Adding template endpoints

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -602,7 +606,59 @@ mailersend.deleteRecipient({
...
+const MailerSend = require("mailersend");
+
+const mailersend = new MailerSend({
+    api_key: "key",
+});
+
+mailersend.deleteTemplate({
+  template_id: 'xxx'
+})
+  .then(response => {
```
Comment (me-shaon) [inline README.md:658]: You can keep it like other endpoints where they are not handling the response at all, like [here](https://github.com/mailersend/mailersend-nodejs/blob/main/examples/webhooks/delete.js#L10). But as you've mentioned it was not the case in the `Domain` endpoint. I think that was a mistake in the example. Later we need to go through all of the existing endpoints and make things consistent everywhere. But for the time being, you can do it like the `Webhook` endpoint.

**Review by me-shaon — CHANGES_REQUESTED** [review]

**Comment by me-shaon** [comment]
@devjavi You need to exclude `.DS_Store` files from the commit. To do that, please add `.DS_Store` to the `.gitignore` file and as you've already added the `.DS_Store` files in your commit, you need to clean that by `git rm --cached .DS_Store` command and commit the changes.

**Comment by devjavi** [comment]
Made changes and also removed the .then() in delete file under examples.

**Review by JuliusB — APPROVED** [review]

File: examples/templates/list.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+"use strict";
+require('dotenv').config()
+
+const MailerSend = require("../../src/MailerSend");
+
+const mailersend = new MailerSend({
+    api_key: process.env.API_KEY,
+});
+
+mailersend.templateList({
```
Comment (me-shaon) [inline examples/templates/list.js:10]: No need to pass an empty object here.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+.DS_Store
```
Comment (me-shaon) [inline .gitignore:3]: I've missed that you've another .DS_Store file in `src/.DS_Store`. Please run `git rm --cached src/.DS_Store` to remove that.

**Review by me-shaon — APPROVED** [review]
I'm approving the PR. Just two minor changes are needed. Fix those and then merge the PR.

**Pull request merged**